### ISSUE TITLE
Loading multiple datasets and distribution for those datasets in one jsonld

### ISSUE THREAD

**Issue by sparsons808** [ckan/ckanext-dcat#226]
does dcat have support for this?

**Comment by metaodi** [comment]
Yes this is possible using the [catalog endpoint](https://github.com/ckan/ckanext-dcat#catalog-endpoint). Just specify `.jsonld` as format.

e.g. https://opendata.swiss/catalog.jsonld

if you only want some datasets, you can use the `q` and `fq` to query/filter the datasets.

**Comment by sparsons808** [comment]
thank you @metaodi , so i know to do this you have to change the ini file, but the documentation doesn't specify where you would place `https://{ckan-instance-host}/catalog.{format}?[page={page}]&[modified_since={date}]&[profiles={profile1},{profile2}]&[q={query}]&[fq={filter query}]` within your code. Also do you know if this end point auto loads into ckan without calling the ckan load api?

**Comment by jze** [comment]
You can read about the catalog endpoint here: https://github.com/ckan/ckanext-dcat#catalog-endpoint

The catalog endpoint is the standard way to harvest data from a data portal that supports DCAT. Unfortunately demo.ckan.org has disabled its CKAN support. However, the catalog endpoint Germany's national open data portal can be found here: https://ckan.govdata.de/catalog.xml
